### DISCUSSION TITLE
Glitch in ragflow

### DISCUSSION THREAD

**Discussion by Naren-NS** [infiniflow/ragflow#2592] [Q&A]
So I have created a knowledge base having a pdf of about 60 pages, when I ask questions to my chatbot the chatbot responds but the response goes away in 1-2 seconds.

Also, is there a way to connect postman to the knowledge base?

**Comment by KevinHuSh** [comment]
Go away in 1-2 seconds?
It means there's a bug.
Have you turn on the TTS?
Could you file an issue to elabrate on it?

**Reply by Naren-NS** [reply]
Yea I guess there is a bug, it also takes very very much time to get a reply from the chatbot. And yes tts is on

**Comment by aamir-gmail** [comment]
All web api are exposed through the web interface, some of them are documented, generate the API key here,
![image](https://github.com/user-attachments/assets/46c0cf0b-2541-4ab6-92ef-5ba85640926b)


and then use this documentation https://github.com/infiniflow/ragflow/blob/main/docs/references/api.md to get started , this is for the postman.

**Reply by KevinHuSh** [reply]
Only API mentioned in the document are exposed.
